.github/workflows/test.sh

```diff
@@ -17,5 +17,6 @@
 
 export CAPNP_PATH="$(pwd)/third_party/capnproto-java/compiler/src/main/schema/"
 export INTERCHANGE_SCHEMA_PATH="$(pwd)/third_party/fpga-interchange-schema/interchange"
+export VIVADO_PATH="$/opt/Xilinx/Vivado/2017.2/bin"
```

Comment:
I think this should be something like `/opt/Xilinx/Vivado/2017.2/bin/vivado`. Or actually it could also be just `vivado` as it gets activated already in here: https://github.com/chipsalliance/f4pga-xc-fasm2bels/blob/master/.github/workflows/check_env.sh#L28-L34

Reply:
I believe that we can try just using `export VIVADO_PATH=vivado`

Reply:
So, I see CI has failed once again, probably vivado needs to be sourced once again. I think we can move this [line](https://github.com/chipsalliance/f4pga-xc-fasm2bels/blob/master/.github/workflows/ci.yml#L93) just before calling the tests.sh script and I think we can get rid of this step: https://github.com/chipsalliance/f4pga-xc-fasm2bels/blob/master/.github/workflows/ci.yml#L90